src/app/components/files/files.component.ts

```diff
@@ -344,4 +344,14 @@ export class FilesComponent extends PageComponent implements OnInit, OnDestroy {
   isFilterHeaderActive(): boolean {
     return this.showHeader === ShowHeaderMode.Always;
   }
+
+  onError(error: any) {
+    if (error?.message) {
```

Comment:
don't think we need to check only 500 error. let's have the template for every error and in the future we can create specific template based on the status if needed

Reply:
Updated